docs/docs/changelog.md

```diff
@@ -1,5 +1,8 @@
 (changelog)=
 
 ```{include} ../../CHANGELOG.md
+Added permutation equivariant DeepSet architecture.
```

Comment:
Not here! In the file in the root directory, named CHANGELOG.md

This one here is only including it in the website but it is not the changelog itself